Packs/BitcoinAbuse/Integrations/BitcoinAbuse/README.md

```diff
@@ -1,5 +1,4 @@
-BitcoinAbuse.com is a public database of bitcoin addresses used by hackers and criminals.
-This integration was integrated and tested with version xx of BitcoinAbuse.
+
```

Comment:
BitcoinAbuse.com is a public database of bitcoin addresses used by hackers and criminals.